01-node-tutorial/answers/16-streams.js

```diff
@@ -0,0 +1,22 @@
+const { createReadStream } = require("fs");
+
+//The program should initialize a counter to 0.???
+
+const stream = createReadStream("../content/big.txt", {
+	highWaterMark: 2000,
+	encoding: "utf8",
+});
+
+let counter = 0;
+stream.on("data", (result) => {
+	counter += 1;
+	console.log(counter);
+});
+stream.on("end", () => {
+	// 	highWaterMark: 2000, 85 chunks received
+	// 	highWaterMark: 200000, 1 chunks received
```

Comment:
The `highWaterMark` is the maximum amount of bytes that Node will read in each chunk of the stream.

Reply:
Thanks. `Test the program for several values of highWaterMark.` was in the assignment. I didn't know how to indicate it